sdk/search/azure-search-documents/README.md

```diff
@@ -32,59 +32,86 @@ create and manage indexes, load data, implement search features, execute queries
+<!-- embedme ./src/samples/java/com/azure/search/documents/ReadmeSamples.java#L62-L65 -->
 ```Java
-private SearchClient searchClient = new SearchClientBuilder().buildClient();
-
-public void createSearchClient() {
-    SearchIndexClient searchIndexClient = new SearchIndexClientBuilder()
+SearchIndexClient searchIndexClient = new SearchIndexClientBuilder()
+    .endpoint(endpoint)
+    .credential(new AzureKeyCredential(apiKey))
+    .buildClient();
 ```
 
 or
 
-<!-- embedme ./src/samples/java/com/azure/search/documents/ReadmeSamples.java#L47-L50 -->
+<!-- embedme ./src/samples/java/com/azure/search/documents/ReadmeSamples.java#L69-L72 -->
 ```Java
-}
-
-public void createAsyncSearchClient() {
-    SearchIndexAsyncClient searchIndexAsyncClient = new SearchIndexClientBuilder()
+SearchIndexAsyncClient searchIndexAsyncClient = new SearchIndexClientBuilder()
+    .endpoint(endpoint)
+    .credential(new AzureKeyCredential(apiKey))
+    .buildAsyncClient();
 ```
 
-#### Create a SearchIndexClient
+#### Create a SearchIndexerClient
 
-To create a SearchIndexClient, you will need an existing index name as well as the values of the Azure Cognitive Search service 
+To create a SearchIndexerClient, you will need an existing index name as well as the values of the Azure Cognitive Search service 
 [URL endpoint](https://docs.microsoft.com/en-us/azure/search/search-create-service-portal#get-a-key-and-url-endpoint) and 
 [query key](https://docs.microsoft.com/en-us/azure/search/search-security-api-keys).
-Note that you will need an admin key to index documents (query keys only work for queries).
+Note that you will need an admin key to do indexer management (query keys only work for queries).
 
-<!-- embedme ./src/samples/java/com/azure/search/documents/ReadmeSamples.java#L54-L58 -->
+<!-- embedme ./src/samples/java/com/azure/search/documents/ReadmeSamples.java#L76-L79 -->
 ```Java
-}
-
-public void createIndexClient() {
-    SearchClient searchClient = new SearchClientBuilder()
-        .endpoint(endpoint)
+SearchIndexerClient searchIndexerClient = new SearchIndexerClientBuilder()
+    .endpoint(endpoint)
+    .credential(new AzureKeyCredential(apiKey))
+    .buildClient();
 ```
 
 or
 
-<!-- embedme ./src/samples/java/com/azure/search/documents/ReadmeSamples.java#L62-L66 -->
+<!-- embedme ./src/samples/java/com/azure/search/documents/ReadmeSamples.java#L83-L86 -->
 ```Java
-}
+SearchIndexerAsyncClient searchIndexerAsyncClient = new SearchIndexerClientBuilder()
+    .endpoint(endpoint)
+    .credential(new AzureKeyCredential(apiKey))
+    .buildAsyncClient();
+```
 
-public void createAsyncIndexClient() {
-    SearchAsyncClient searchAsyncClient = new SearchClientBuilder()
-        .endpoint(endpoint)
+#### Create a SearchClient
+
+Once you have the values of the Azure Cognitive Search service [URL endpoint](https://docs.microsoft.com/en-us/azure/search/search-create-service-portal#get-a-key-and-url-endpoint) 
```

Comment:
Should mention for `SearchClient` that you need the name of an index that has already been created.